src/modules/commander/commander_params.c

```diff
@@ -977,3 +977,15 @@ PARAM_DEFINE_INT32(COM_PREARM_MODE, 1);
  * @group Commander
  */
 PARAM_DEFINE_INT32(COM_MOT_TEST_EN, 1);
+
+/**
+ * Enable immediate disarm on kill
+ *
+ * If set, engaging the kill switch will immediately disarm the vehicle. Otherwise
+ * the vehicle disarms after 5 seconds.
+ *
+ * @boolean
+ * @reboot_required true
+ * @group Commander
+ */
+PARAM_DEFINE_INT32(COM_KILL_DISARM, 0);
```

Comment:
Any reason you'd want a time between zero and 5s ? If so, maybe a time like all the other disarm parameters?

Reply:
Yeah I'm guna change it to that. Blame @dagar :laughing:

Reply:
I always do :-). But it was a question - if the time is always going to be  5 or off then boolean is fine.